### PR TITLE
Doc cleanup

### PR DESCRIPTION
**Pull request by AnthonyGiorgio**
I fixed a variety of spelling, grammar, and formatting issues in some of the Markdown files.  I also resolved a bunch of warnings from the Markdown linter.

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
Thanks, I'll leave some comments inline.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This is continuation of venerable JA2-Stracciatella project.
 
 The goal of the project is to make Jagged Alliance 2 available on a wide range of platforms, improve its stability, fix bugs and provide a stable platform for mod development.  At the moment the goal is mostly to fix bugs.
 
-Official Homepage: http://ja2-stracciatella.github.io/
+[Official Homepage](http://ja2-stracciatella.github.io)
```
Comment (lynxlynxlynx) [inline README.md:12]: This is not good for two reasons: the URL is not visible any more and the line stands out. The intent is to lay it out, so if the linter is complaining about the bare URL, just add the (reduntant) URL syntax around it with itself as the text.

Comment (AnthonyGiorgio) [reply README.md]: I've reverted the change.

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**